src/MaciVoting.sol

```diff
@@ -268,7 +243,7 @@ contract MaciVoting is PluginUUPSUpgradeable, ProposalUpgradeable, IMaciVoting {
             startDate: _startDate,
             endDate: _endDate,
             treeDepths: treeDepths,
-            messageBatchSize: 20,
+            messageBatchSize: messageBatchSize,
```

Comment:
More of a question than a suggestion, do we want to also abstract the following args here at some point? Or are they ok hardcoded?
1. `mode: DomainObjs.Mode.NON_QV,`
2. `voteOptions: 3`

Reply:
I was thinking on leaving these hardcoded but it would be better to take them out as env variables :)

1. we should use `DomainObjs.Mode.FULL` now (I will change it)
2. `voteOptions`: yes, no, abstain (I don't think DAOs need more but just in case lets use a env variable here)